gcloud-java-contrib/README.md

```diff
@@ -0,0 +1,59 @@
+Google Cloud Java Contributions
+===============================
+
+Additional functionality for gcloud-java (Java idiomatic client for [Google Cloud Platform][cloud-platform] services).
+
+Quickstart
```

Comment:
This section probably should be commented out until we push a release to Maven.  (If a user tried the following, then they would get an error until we do a release).